### DISCUSSION TITLE
message queuing

### DISCUSSION THREAD

**Discussion by frogg1ven** [jgromes/RadioLib#475] [General]
hi, I would like to know if there is any possibility to queue incoming messages to lora, because now if I send two messages one after another without big delay, only first one comes.
Im using lora e220 for sending and e22 for receiving, with yours modified interrupt examples.

**Comment by jgromes** [comment]
This is not something that's supported by the library, since different platforms might have to use different queuing systems, so it's up to the user to implement.

**Reply by frogg1ven** [reply]
Thanks for reply
